frontend/lib/identification/id_card.dart

```diff
@@ -3,19 +3,20 @@ import 'package:flutter/rendering.dart';
 
 class IdCard extends StatelessWidget {
   final Widget child;
+  final double height;
 
-  const IdCard({Key key, this.child}) : super(key: key);
+  const IdCard({Key key, this.child, this.height}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return IntrinsicHeight(
-        child: FittedBox(
-            child: Card(
-                shape: RoundedRectangleBorder(
-                  borderRadius: BorderRadius.circular(10.0),
-                ),
-                clipBehavior: Clip.antiAliasWithSaveLayer,
-                color: Colors.black26,
-                child: child)));
+    return Card(
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(10.0),
+        ),
+        clipBehavior: Clip.antiAliasWithSaveLayer,
+        color: Colors.black26,
+        child: Container(
+            height: height,
```

Comment:
Why do we force a specific height here?

Reply:
In landscape mode the card doesn't fill the whole width, so its necessary to either specify the height or the width. It wasn't as easy as I thought to have relative measurements (like expanding to 50% width) in this widget.